.gitignore

```diff
@@ -23,3 +23,6 @@ npm-debug.log
 
 # Ignore bundle dependencies
 vendor/ruby
+
+# RVM gemset
+.ruby-gemset
```

Comment:
I keep all gems in isolated gemsets for two reasons:

1. I maintain `yo` generator, so I scaffold a lot for testing purposes. It's easier for me to delete scaffolded test gems by deleting gemset.
1. To sleep easily.